website/docs/configuration/modules.html.md

```diff
@@ -191,38 +235,14 @@ from a module can be referenced from elsewhere in your configuration.
 When refactoring an existing configuration to introduce modules, moving
 resource blocks between modules causes Terraform to see the new location
 as an entirely separate resource to the old. Always check the execution plan
-after performing such actions to ensure that no resources are surprisingly
-deleted.
-
-## Other Meta-arguments
-
-Along with the `source` meta-argument described above, module blocks have
-some more meta-arguments that have special meaning across all modules,
-described in more detail in other sections:
-
-* `version` - (Optional) A [version constraint string](./version-constraints.html)
-  that specifies acceptable versions of the module. Described in detail above.
-
-* `providers` - (Optional) A map whose keys are provider configuration names
-  that are expected by child module and whose values are corresponding
-  provider names in the calling module. This allows
-  [provider configurations to be passed explicitly to child modules](#passing-providers-explicitly).
-  If not specified, the child module inherits all of the default (un-aliased)
-  provider configurations from the calling module.
+after performing such actions to ensure that no resources are deleted by
+surprise; if necessary, use [the `terraform state mv`
```

Comment:
Nice addition. I feel the advice regarding the usage of `terraform state mv` is lacking when reading about refactoring; this is a nice start.

Reply:
Yeah, I think I meant to come back to that and spaced.